### PR TITLE
exclude nans channels from check for nans

### PR DESCRIPTION
**Pull request by romainmartinez**
## Proposed changes

We have a function to ensure that the user does not send channels containing nans in our signal processing functions (`check_for_nans`). I just added a quick fix to exclude channels that are tagged as nans from the function described above.

## Types of changes

What types of changes does your code introduce?

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging your code._

- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: pyomeca/frame_dependent.py [hunk 1]
```diff
@@ -562,7 +562,12 @@ def check_for_nans(self, threshold_channel=10, threshold_consecutive=5):
             Threshold of tolerated consecutive nans on each channel
         """
         # check if there is nans
-        nans = np.isnan(self)
+        if self.get_nan_idx is not None:
```
Comment (pariterre) [inline pyomeca/frame_dependent.py]: Is this tag set for sure?

Comment (romainmartinez) [reply pyomeca/frame_dependent.py]: Yes, it is part of the `__init__` of the FrameDependent class:

```python
self.get_nan_idx = None
```

**Comment by pariterre** [comment]
Hi,

Can you have a look at this "use-local" by travins?

**Comment by romainmartinez** [comment]
We just had to add the conda-forge channel in travis & appveyor

**Pull request merged**